docs/setup-auto-instrumentation.asciidoc

```diff
@@ -120,19 +120,18 @@ can be used as part of a https://docs.docker.com/develop/develop-images/multista
 
 [source,sh]
 ----
-ARG AGENT_VERSION=1.11.1
+ARG AGENT_VERSION=1.19.0
 
 FROM alpine:latest AS build
 ARG AGENT_VERSION
 WORKDIR /source
 
 # install unzip
-RUN apt-get -y update && apt-get -yq install unzip
+RUN apk update && apk add zip curl
 
 # pull down the zip file based on ${AGENT_VERSION} ARG and unzip
-RUN curl -L -o elastic_apm_profiler_${AGENT_VERSION}.zip https://github.com/elastic/apm-agent-dotnet/releases/download/${AGENT_VERSION}/elastic_apm_profiler_${AGENT_VERSION}.zip && \ 
+RUN curl -L -o elastic_apm_profiler_${AGENT_VERSION}.zip https://github.com/elastic/apm-agent-dotnet/releases/download/v${AGENT_VERSION}/elastic_apm_profiler_${AGENT_VERSION}.zip && \ 

```

Comment:
Ahh, nice!

At some point we added `v` before the version numbers in the .NET agent, in order to be aligned with other APM agents. So the url without `v` worked for older versions, but it's now needed here for newer versions.